main_functional_test.go

```diff
@@ -216,6 +218,108 @@ var _ = Describe("Functional", func() {
 			})
 		})
 	})
+
+	Describe("AWS SQS", func() {
+
+		// Get secrets somehow
+		var queueName string
+
+		BeforeEach(func() {
+			queueName = fmt.Sprintf("FunctionalTestQueue%d", rand.Int())
+			err := createSqsQueue(queueName)
+			if err != nil {
```

Comment:
Any reason why not just `Expect(err).ToNot(HaveOccurred())`?

Reply:
Good call, fixed